wgpu-core/src/command/compute.rs

```diff
@@ -84,6 +102,7 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
         let raw = cmb.raw.last_mut().unwrap();
         let mut binder = Binder::new(cmb.limits.max_bind_groups);
 
+        let (_, mut token) = hub.render_bundles.read(&mut token);
```

Comment:
Why do we need to grab a token for something we don't use?

Reply:
yeah, this is due to the type-level deadlock protection that we have: you can't lock anything after anything. There is a graph of possible locks defined in `hub.rs`. Sometimes I feel that it's easier to lock an intermediate step (for read, of course), instead of modifying the graph. Worried that if it becomes too complex, it will be difficult to find bugs in it...

Reply:
I feel like this problem could be solved with some kind of build script to generate this or something. As with A -> B -> C, A -> C should be perfectly safe. Discussion for another time :)

Reply:
yep, I'd love to discuss this on the matrix :)